index.rst

```diff
@@ -575,6 +579,8 @@ Adding a new Conda package
 Making a Manual Release
 =======================
 
+XXX I'm a little uncomfortable even writing this isn the release process readme.  It seems like a see `link` for how the sausage is made situation, if at all. XXX
+
```

Comment:
OK - so remove everything after the link to jenkins?  This section is already somewhat out of date.